client/app/main/student/student.controller.js

```diff
@@ -14,6 +14,16 @@ function StudentCtrl($scope, $state, $stateParams, Student, toastr, Modal) {
     $scope.student = result.student;
     $scope.interventions = result.interventions;
     $scope.outreaches = result.outreaches;
+    Student.list({id: $stateParams.id}, function(data) {
+      var record = data.currentRecord.entries;
+      var enrolled = 0;
+      var present = 0;
+      _.forEach(record, function(entry) {
```

Comment:
I'm pretty sure we want a decimal since our break point is at the whole number. Otherwise the display and the categorization could be conflicting.